### PR TITLE
EZP-31980: New design for notifications/alerts

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31980
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Screen:
Zrzut ekranu 2020-10-5 o 14 49 53

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig [hunk 1]
```diff
@@ -13,103 +13,141 @@
...
         {% set badge = 'Trial' %}
         {% if ez.isEndOfLife %}
             {% set severity = 2 %}
-            <div class="alert alert-danger mb-0 mt-3" role="alert">
+            <div class="alert alert-danger mb-0 mt-3 ez-alert ez-alert--icon" role="alert">
+                <svg class="ez-icon ez-icon--small-medium ez-icon--light">
+                    <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#information"></use>
+                </svg>
+                <span>
                 {{ 'dashboard.ez_version.trial_expired'|trans({'%ttl_url%': urls['ttl']})|desc("Unfortunately your trial period has expired and your <a target=\"_blank\" href=\"%ttl_url%\">TTL license</a> is no longer valid.")|raw }}
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:55]: Should here be extra indent? You added <span> above

File: src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig [hunk 2]
```diff
@@ -13,103 +13,141 @@
...
-                |desc("Unfortunately %release% open source version has reached end of life,
-                    <a target=\"_blank\" href=\"%update_url%\">please upgrade</a>.")
-                |raw }}
-                <em>
-                    {{ 'dashboard.ez_version.community_end_of_maintenance_upgrade'|trans({
-                        '%license_url%': urls['license'],
-                        '%ee_product_url%': urls['ee_product'],
-                        '%support_service_url%': urls['support_service'],
-                        '%service_life_url%': urls['service_life']
+            <div class="alert alert-warning mb-0 mt-3 ez-alert ez-alert--icon" role="alert">
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:87]: Shouldn't here be extra ident? At the moment it's at the same level as `{% if %}`

File: src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig [hunk 3]
```diff
@@ -13,103 +13,141 @@
...
-            <div class="alert alert-info mb-0 mt-3" role="alert">
-                {{ 'dashboard.ez_version.community_severity_non'|trans({
-                    '%release%': ez.release,
-                    '%license_url%': urls['license'],
-                    '%gpl_faq_url%': urls['gpl_faq'],
-                })
-                |desc("Welcome to the open source %release% release. Using the community friendly <a target=\"_blank\" href=\"%license_url%\">GPL license</a>,
-                    <a target=\"_blank\" href=\"%gpl_faq_url%\">sharing your code</a> is what it's all about.")
-                |raw }}
+            <div class="alert alert-info mb-0 mt-3 ez-alert ez-alert--icon" role="alert">
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:117]: As above

File: src/bundle/Resources/views/themes/admin/ui/notification.html.twig [hunk 4]
```diff
@@ -1,8 +1,11 @@
 <div class="alert alert-{{ label }} alert-dismissible fade show">
     <button type="button" class="close" data-dismiss="alert" aria-label="Close" title="{{ 'alert.close'|trans|desc('Close') }}">
-        <svg class="ez-icon ez-icon--light ez-icon-small">
-            <use xlink:href="/bundles/ezplatformadminui/img/ez-icons.svg#circle-close"></use>
+        <svg class="ez-icon ez-icon--light ez-icon--small">
+            <use xlink:href="/bundles/ezplatformadminui/img/ez-icons.svg#discard"></use>
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/notification.html.twig:4]: `asset()`

File: src/bundle/Resources/views/themes/admin/ui/notification.html.twig [hunk 5]
```diff
@@ -1,8 +1,11 @@
...
         </svg>
     </button>
-    {{ message }}
+    {% if badge is defined %}
+    <span class="badge ez-badge ez-badge--small ez-badge--round">{{ badge }}</span>
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/notification.html.twig:8]: indent

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig (hunk 2 above)
Comment (dew326) [reply src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:87]: I think it is not at the same level as` if`

File: src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig (hunk 3 above)
Comment (dew326) [reply src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:117]: also seems correct

**Review by lucasOsti — APPROVED** [review]

**Review by lserwatka — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Pull request merged**